### DISCUSSION TITLE
为什么不能通过扫码登录B站账号呢？

### DISCUSSION THREAD

**Discussion by GAKKI-OuO** [leiurayer/downkyi#921] [Q&A] [answered]
打开扫码界面会自动显示登录成功，然后回到软件主页。无论扫没扫码都是这样，即使在B站APP上确认登录，也没用，这是为啥呢？系统是Win11，而且我尝试下载了不同的版本，都是相同的问题。

在网上搜，发现目前有不少人面临此问题，且无从解决。

请大佬解惑！谢谢🙏

**Comment by yaobiao131** [comment, marked as answer]
https://github.com/leiurayer/downkyi/issues/905#issuecomment-1827937695

**Reply by GAKKI-OuO** [reply]
搞定了，谢谢你🙏
